packages/core/saved-objects/core-saved-objects-base-server-internal/src/migration/kibana_migrator.ts

```diff
@@ -65,6 +65,7 @@ export interface MigrateDocumentOptions {
    * Defaults to `false`.
    */
   allowDowngrade?: boolean;
+  downwardConversion?: 'lax' | 'forbid';
```

Comment:
I think this can be removed? At least I don't see it being used in the implementation.